tests/test_session.py

```diff
@@ -81,6 +82,24 @@ def testRedirect(self):
         self.assertEqual(b.data, b"123")
 
 
+class DiskStoreTest(unittest.TestCase):
+    def testStoreConcurrent(self):
+        dir = tempfile.mkdtemp()
+        store = web.session.DiskStore(dir)
+
+        def set_val():
+            store["fail"] = "value"
+
+        for c in range(10):
+            m = threading.Thread(target=set_val)
+            m.start()
+            try:
+                value = store["fail"]
+            except KeyError:
+                pass
+        self.assertEquals(value, "value")
```

Comment:
__assertEquals()__ (with a trailing '__s__') was removed in Python 3 so please use __assertEqual()__ instead.